src/test/java/com/artipie/asto/cache/FromStorageCacheTest.java

```diff
@@ -137,22 +137,26 @@ void processMultipleRequestsSimultaneously() throws Exception {
         final FromStorageCache cache = new FromStorageCache(this.storage);
         final Key key = new Key.From("key4");
         final int count = 100;
-        final CountDownLatch latch = new CountDownLatch(10);
+        final CountDownLatch latch = new CountDownLatch(
+            Runtime.getRuntime().availableProcessors() - 1
```

Comment:
@andreoss what if any runtime has only one CPU, I think some CI isntances may have 1 CPU per builder?

Reply:
@g4s8 The latch will be no-op than. The test relys on common fork pool as I understand, and won't make much sense on a single-threaded machine